sig/datadog/profiling/scheduler.rbs

```diff
@@ -0,0 +1,19 @@
+module Datadog
+  module Profiling
+    class Scheduler < Core::Worker
+      include Core::Workers::Polling
+
+      def initialize: (
+        exporter: Datadog::Profiling::Exporter,
+        transport: Datadog::Profiling::HttpTransport,
+        ?fork_policy: untyped,
```

Comment:
Sometimes for `untyped` things I create a type alias, e.g:

```
type fork_policy = untyped

def initialize:
  ...
  ?fork_policy: fork_policy
```

Allowing me to (and Steep) to differentiate between a blanket `uptyped` and two "different-but-known" kinds of `untyped`, and then if/when I get to actually type it, I can replace it in one place.

WDYT?